RELEASING.md

```diff
@@ -26,6 +26,13 @@ otherwise git might convert line endings in some cases.
 Make sure you have the Lightbend Whitesource credentials configured in
 your `~/.sbt/1.0/private-credentials.sbt`.
 
+## Snapshot releases
+
+Nightly snapshot releases are created from master and published to
+https://repo.akka.io/snapshots by https://jenkins.akka.io:8498/job/akka-publish-nightly/
+
+To create snapshot versions manually, use `sbt clean stampVersion publish`.
```

Comment:
is this correct? doesn't it miss the rsync part? I have been using this job when publishing from wip branches: https://jenkins.akka.io:8498/job/akka-publish-wip/